daemon/nextbox-compose/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM --platform=linux/aarch64 nextcloud:27.1.6-apache
+
+ENV DEBIAN_FRONTEND noninteractive
+
+RUN apt-get -y update && \
+    apt-get -y install --no-install-recommends \
+        smbclient
```

Comment:
could we also use the php install method for an `smbclient` variant ?

Reply:
Sadly no, i thought we could, but not with `docker-php-ext-install`, as it only ships a selection of php modules, which does not contain `smbclient`

Reply:
ok, then we'll likely need to do the full `NON_INTERACTIVE` and friends stuff here to avoid any issues based on `apt` waiting for interactions  (+ cleaning up whatever `update` generated) ... maybe this is also connected to why this won't work on debian 10...